pkg/apis/network/v1/pki_types.go

```diff
@@ -0,0 +1,60 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PKISpec defines the desired state of PKI.
+// +k8s:openapi-gen=true
+type PKISpec struct {
+	// caCert is information about the CA certificate and key.
+	CACert CertSpec `json:"caCert"`
+
+	// caBundleName is the name of the config-map containing the CA bundle.
+	CABundleName string `json:"caBundleName"`
```

Comment:
What format is the config-map in? Why is this needed? If you're creating a CA then why do we need to provide a CA bundle? Or do you mean the newly-created CA gets published to the object named here? Why not just require the `ConfigMap` to have the same name as the `PKI`?

Reply:
good point. hard-coding the names is easier.